api/src/Entity/Period.php

```diff
@@ -245,4 +247,24 @@ public function removeMaterialItem(MaterialItem $materialItem): self {
 
         return $this;
     }
+
+    /**
+     * The day number of the first Day in period.
+     */
+    public function getFirstDayNumber(): int {
```

Comment:
Wäre ein Unit Test dafür möglich? Wenn du `$this->camp->periods` mockst könnte es gehen oder?